roles/ceph-mon/tasks/docker/start_docker_monitor.yml

```diff
@@ -10,6 +10,43 @@
 - name: pull ceph daemon image
   shell: "docker pull {{ ceph_mon_docker_username }}/{{ ceph_mon_docker_imagename }}"
 
+# Use systemd to manage container on Atomic host
+- name: generate systemd unit file
+  sudo: true
+  config_template:
```

Comment:
I don't think `config_template` is needed here (unless you want users to be able to add more variables to the unit file).
Simply use the `template` module.